README.md

```diff
@@ -26,6 +26,7 @@ Do some configuration
 config :cached_contentful,
     space_id: "YOUR_SPACE_ID",
     access_token: "YOUR_ACCESS_TOKEN",
+    environment: "YOUR_ENVIRONMENT",
```

Comment:
Does this mean we have to set env variables in the container of are they used from the config.exs files? The latter would have my preference.

Reply:
Or is this dependant on the configuration in the "parent" application?

Reply:
This is all from the config file and needs to be set in the child application

Reply:
Ok so in the case of villa-augustus, this config variable wil be set in the config files of that application?